google/cloud/spanner_dbapi/connection.py

```diff
@@ -116,6 +116,7 @@ def __init__(self, instance, database=None, read_only=False):
         self._batch_mode = BatchMode.NONE
         self._batch_dml_executor: BatchDmlExecutor = None
         self._transaction_helper = TransactionRetryHelper(self)
+        self._schema_name = database.default_schema_name
```

Comment:
(Overall comment, not specifically for this line): You should always add tests, even for something as simple as this. Not because it is unclear whether this implementation works as expected, but to prevent accidental regressions in the future. In this case, we should verify that the returned value for `current_schema` is "" for GoogleSQL and "public" for PostgreSQL. This prevents accidental regressions if someone modifies the property definition in the future without realizing that other features might depend on this specific behavior.

Reply:
Ack. Will take care from next time.
Done